client/src/components/ItemDetail/ItemDetail.tsx

```diff
@@ -63,13 +83,31 @@ function ItemDetail({
     }
     // Everytime we change to a 'new' item, refresh removal state flag
     setIsRemoving(false);
+    // Refresh whether we collapse media (image) to show more of text
+    setMediaCollapsed(false);
+    // Determine whether we should render 'show more' expand button
+    const contentDiv = document.getElementById("item-detail-description");
+    if (contentDiv) setIsOverflow(checkContentOverflow(contentDiv));
+    else setIsOverflow(false); // Cannot find content tag, no content to render
+    // Compute the margin required to offset minHeight for footer
+    const footerMarginOffset =
+      footerMinHeight -
+      document.documentElement.clientHeight * footerPercentageHeight;
+    if (footerMarginOffset > 0)
+      setMinHeightMarginTop(
+        Math.floor(
+          (100 * footerMarginOffset) / document.documentElement.clientHeight
+        )
+      );
+    else setMinHeightMarginTop(0);
   }, [cartItem]);
 
   return (
     <div style={{ height: "100vh", width: "100%" }}>
       <IconButton
         id="item-detail-dismiss"
         style={{
+          zIndex: 1004,
```

Comment:
Why this high z-index?

Reply:
I arbitrarily chose 1000 as a starting point, in the future, we would want to have some kind of dynamic z-index manager for layering UI controls #198